src/pykeen/contrib/lightning.py

```diff
@@ -150,13 +150,15 @@ def _dataloader(self, triples_factory: CoreTriplesFactory, shuffle: bool = False
         """Create a data loader."""
         raise NotImplementedError
 
-    def train_dataloader(self):
+    def train_dataloader(self) -> torch.utils.data.DataLoader:
         """Create the training data loader."""
         return self._dataloader(triples_factory=self.dataset.training, shuffle=True)
 
-    def val_dataloader(self):
+    def val_dataloader(self) -> OneOrSequence[torch.utils.data.DataLoader]:
```

Comment:
Can it be `Optional[torch.utils.data.DataLoader]` instead? Seems kind of strange to make a sequence only when it's empty. Or is that how lightning expects it?

Reply:
It's defined by PyLightning.

Reply:
okay, thanks for the clarification!